android/src/fabric/java/com/reactnativepagerview/LEGACY_PagerViewViewManager.kt

```diff
@@ -0,0 +1,16 @@
+package com.reactnativepagerview
+
+import com.facebook.react.uimanager.ViewGroupManager
+import android.widget.FrameLayout
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.ThemedReactContext
+
+
+@ReactModule(name = "LEGACY_RNCViewPager")
```

Comment:
Can we use a shared variable between `LEGACY_` fabric and paper for the name? Same as we did for normal implementation: https://github.com/callstack/react-native-pager-view/blob/e5a2f7e70e030209a86a2d4434fd8159c50dc635/android/src/fabric/java/com/reactnativepagerview/PagerViewViewManager.kt#L22C21-L22C21

Reply:
@okwasniewski done, wasn't sure where to extract it best so let me know what you think please, thanks